packages/block-library/src/buttons/editor.scss

```diff
@@ -1,6 +1,6 @@
-.wp-block-buttons .wp-block.block-editor-block-list__block[data-type="core/button"] {
-	display: inline-block;
-	width: auto;
+.wp-block-buttons > .wp-block {
+	// Override editor auto block margins.
+	margin-left: 0;
```

Comment:
Curious why this changed so much though?

Reply:
I found that both the `display` and `width` properties were unnecessary. The former was already set by `style.scss`, and the latter was already `auto` by default. (And the editor never overrode it.) I tested with various alignments to make sure everything worked as expected.

I also found that the selector had a lot more specificity than it actually needed, so I simplified that. Notably, I chose to point at just the `wp-block` class for the children, to make it easier for 3rd-party button blocks to be inserted and styled correctly in the Buttons block. I almost just went with a `.wp-block-buttons > *` selector, but that turned out to not have enough specificity to override the `margin-left` value being set by the editor in some cases.